tool/tsh/db.go

```diff
@@ -505,7 +506,43 @@ func WithLocalProxy(host string, port int, caPath string) ConnectCommandFunc {
 	}
 }
 
-func getConnectCommand(cf *CLIConf, tc *client.TeleportClient, profile *client.ProfileStatus, db *tlsca.RouteToDatabase, opts ...ConnectCommandFunc) (*exec.Cmd, error) {
+// execer is an abstraction of Go's exec module, as this one doesn't specify any interfaces.
+// This interface exists only to enable mocking.
+type execer interface {
+	// RunCommand runs a system command.
+	RunCommand(name string, arg ...string) ([]byte, error)
+	// LookPath returns a full path to a binary if this one is found in system PATH,
+	// error otherwise.
+	LookPath(file string) (string, error)
+}
+
+// systemExecer implements execer interface by using Go exec module.
+type systemExecer struct{}
+
+// RunCommand is a wrapper for exec.Command(...).Output()
+func (s systemExecer) RunCommand(name string, arg ...string) ([]byte, error) {
+	return exec.Command(name, arg...).Output()
+}
+
+// LookPath is a wrapper for exec.LookPath(...)
+func (s systemExecer) LookPath(file string) (string, error) {
+	return exec.LookPath(file)
+}
+
+type cliCommandBuilder struct {
```

Comment:
What do you think about moving all command-building-related stuff (builder, execer, constants, etc.) into a separate file like `dbcmd.go` or `dbexec.go`?

Reply:
Moved into `dbcmd.go`.